lib/BloqadeSchema/src/parse.jl

```diff
@@ -52,13 +54,16 @@ function discretize_with_warn(wf::Waveform,warn::Bool,max_slope::Real,min_step::
             )
             return new_wf
         else
-            throw(e)
+            error(e.msg)
```

Comment:
```suggestion
            rethrow(e)
```

we need to keep the stack trace